addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -466,22 +466,20 @@ options.registry.WebsiteSaleProductPage = options.Class.extend({
     },
 
     /**
-     * If an accordion is displayed in the product details, remove the classes
-     * that bootstrap adds when editing an accordion-item to avoid saving the
-     * template with the accordion opened.
+     * If an accordion is displayed in the product details, remove the classes that Bootstrap adds
+     * when editing an accordion-item to avoid saving the template with the accordion opened.
      *
      * @override
      */
     async cleanForSave() {
         const accordionEl = this.productDetailMain.querySelector('#product_accordion');
         if (!accordionEl) return;
-        const accordionItemsEls = accordionEl.querySelectorAll('.accordion-item');
 
-        accordionItemsEls.forEach((item, key) => {
+        const accordionItemsEls = accordionEl.querySelectorAll('.accordion-item');
+        accordionItemsEls.forEach(item, key => {
```

Comment:
Removing these parenthesis would break the forEach as it can't access its parameters. We need them 😄